.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+## Description
```

Comment:
I think if you just name this file `.github/PULL_REQUEST_TEMPLATE.md` it will work the same way. The folder is only if you have multiple templates.

Reply:
Wasn't sure but I have made the changes.